asrtoolkit/data_structures/audio_file.py

```diff
@@ -26,19 +29,17 @@ def cut_utterance(source_audio_file,
     end_time: float or str
     sample_rate: int, default 16000; audio sample rate in Hz
 
-    uses sox segment source_audio_file to create target_audio_file that contains audio from start_time to end_time
+    uses sox to segment source_audio_file to create target_audio_file that contains audio from start_time to end_time
         with audio sample rate set to sample_rate
     """
     subprocess.call(
-        [
-            "sox {} -r {} -b 16 -c 1 {} trim {} ={}".format(
-                source_audio_file,
-                str(sample_rate),
-                target_audio_file,
-                str(start_time),
-                str(end_time),
-            )
-        ],
+        "sox -V1 {} -r {} -b 16 -c 1 {} trim {} ={}".format(
+            source_audio_file,
+            str(sample_rate),
+            target_audio_file,
+            str(start_time),
+            str(end_time),
```

Comment:
I don't think the `str` calls are necessary here? Format should handle that for us.